packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -532,8 +532,8 @@ export class MarketOperationUtils {
         };
 
         // NOTE: For sell quotes input is the taker asset and for buy quotes input is the maker asset
-        const takerAssetToEthRate = side === MarketOperation.Sell ? ethToInputRate : ethToOutputRate;
-        const makerAssetToEthRate = side === MarketOperation.Sell ? ethToOutputRate : ethToInputRate;
```

Comment:
@kimpers Kim, I think you named these variables, does it make sense to rename them? I think it should be `ethToXAssetRate`, correct?

Reply:
There were a lot of discussions back and forth about whether `ethToXAssetRate` or `XAssetToEthRate` is the correct naming 😅  I'm open to changing it but maybe we should just have a vote to get a consensus on what makes the most sense? 🤔 

Another thing to consider is that we now have `sellTokenToEthRate` and `buyTokenToEthRate` in the quote response, so if we change this we may or may not want to change that as well (could have implications for integrators if anyone is relying on that value).

Reply:
I don't see how the original naming was correct. E.g., in a sell:
- `taker == input`
- `maker == output`
- so `ethToInputRate == ETH -> taker`, not `taker -> ETH`
??

Reply:
> I don't see how the original naming was correct. E.g., in a sell:
> 
> * `taker == input`
> * `maker == output`
> * so `ethToInputRate == ETH -> taker`, not `taker -> ETH`
>   ??

I dug into the old relevant PRs (for ref 0xProject/protocol#49 and 0xProject/0x-api/pull/433).

The problem here is (and correct me if I'm wrong) that `ethToInputRate` and `ethToOutputRate` are named in a funky way ([See discussions](https://github.com/0xProject/0x-api/pull/433#discussion_r534613501), and it should have been cleaned up but wasn't.

Let's use an example sell: 1000 USDC for ETH: https://api.0x.org/swap/v1/quote?sellAmount=1000000000&sellToken=USDC&buyToken=ETH.

In this case `sellTokenToEthRate   == takerAssetToEthRate == ethToInputRate`, the value is `1350.879851 USDC`. So `ethToInputRate` is actually the rate of for going takerToken to ETH. Or am I missing something? 🤔

Reply:
I think this is a case where two wrongs make a right. Perhaps the computation in API undoes this weirdness (so that would break), but `ethToInputRate` and `ethToOutputRate` appear to be getting populated correctly earlier on from what I can see.

```ts
            // Get ETH -> makerToken token price.
            this._sampler.getMedianSellRate(feeSourceFilters.sources, makerToken, this._wethAddress, ONE_ETHER),
            // Get ETH -> taker token price.
            this._sampler.getMedianSellRate(feeSourceFilters.sources, takerToken, this._wethAddress, ONE_ETHER),
...
    public getMedianSellRate(
        sources: ERC20BridgeSource[],
        makerToken: string,
        takerToken: string,
        takerFillAmount: BigNumber,
    ): BatchedOperation<BigNumber> {

```

Reply:
We had a long discussion about this after Thursday's daily and came to the conclusion that both namings are in fact correct they just have a different meaning of the word `to`.

In `ethToInputRate` the `to` implies a rate to multiply with, in `takerAssetToEthRate` the `to` implies takerAsset over eth. The conclusion is that we should clean this up and rename the variables in a less ambiguous way 👍

Reply:
imo `to` is always multiply

Reply:
Thanks Kim, @dorothy-zbornak we want to rename these on the asset-swapper side to `takerAssetPriceForOneEth` to remove all ambiguity